### ISSUE TITLE
install time is just absurd

### ISSUE THREAD

**Issue by isaacimholt** [magarena/magarena#1422]
![image](https://user-images.githubusercontent.com/12482837/35482106-9bad4a62-0430-11e8-9078-6159c7001029.png)

It's all those single .txt files that take forever to copy. Please use a database or put them all into a single file or something. This is just insanity.

**Comment by melvinzhang** [comment]
Thanks for letting us know about this, the included gif is helpful to illustrate the problem. Unfortunately, it doesn't provide any info on how to replicate the issue on our end.

Please let us know the steps you took after downloaded the zip file to your computer. Based on my best guess, it could be something like the following:
1. Download zip to disk.
2. Double click the zip in Windows explorer
3. Drag the folder from inside the zip to another folder on disk.

Is this what you tried to do?

**Comment by isaacimholt** [comment]
Yes that's exactly what I did. In particular I was moving the files to an external hard drive, so the time to move the files was even longer. However this happens also when using my main hard-drive and while it is faster, it is still agonizingly slow.
